### PR TITLE
Pcornet v3.1

### PR DESCRIPTION
**Pull request by dcam2015**
@bruth Added version 3.1 of the pcornet model.

### CODE REVIEW

File: pcornet/v3.1/constraints/primary_keys.csv [hunk 1]
```diff
@@ -16,7 +16,6 @@ pcornet,v3.1,primary key,pcornet_trial,patid,xpk_pcornet_trial
 pcornet,v3.1,primary key,pcornet_trial,trialid,xpk_pcornet_trial
 pcornet,v3.1,primary key,pcornet_trial,participantid,xpk_pcornet_trial
 pcornet,v3.1,primary key,death,patid,xpk_death
-pcornet,v3.1,primary key,death,death_date,xpk_death
```
Comment (dcam2015) [inline pcornet/v3.1/constraints/primary_keys.csv]: @bruth Could you confirm this change when you can?  According to page 70 of the [specs ](http://pcornet.org/wp-content/uploads/2016/11/2016-11-15-PCORnet-Common-Data-Model-v3.1_Specification.pdf),  it was removed as a constraint.

Comment (bruth) [inline pcornet/v3.1/constraints/primary_keys.csv]: Yep that looks correct.

File: pcornet/v3.1/constraints/not_nulls.csv [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+pcornet,v3.1,not null,pro_cm,pro_item,
+pcornet,v3.1,not null,pro_cm,pro_date,
+pcornet,v3.1,not null,pro_cm,pro_response,
+pcornet,v3.1,not null,prescribing,prescribingid,
+pcornet,v3.1,not null,prescribing,patid,
+pcornet,v3.1,not null,pcornet_trial,patid,
+pcornet,v3.1,not null,pcornet_trial,trialid,
+pcornet,v3.1,not null,pcornet_trial,participantid,
+pcornet,v3.1,not null,death,patid,
+pcornet,v3.1,not null,death,death_date,
```
Comment (bruth) [inline pcornet/v3.1/constraints/not_nulls.csv]: This constraint needs to be removed as well since `death_date` can be null.

Comment (dcam2015) [inline pcornet/v3.1/constraints/not_nulls.csv]: @bruth Updated

**Pull request merged**